config/initializers/feature_toggles.rb

```diff
@@ -6,10 +6,6 @@
   if beta_testers == ['all']
     true
   else
-    beta_testers.include?(user.email)
+    beta_testers.include?(user&.email)
```

Comment:
This is very likely to cause merge conflicts with the PR I'm currently working on. Besides, I don't understand why it would be needed.

Reply:
Your implementation called the feature toggle block only when `user.present?`. But that meant that the block wasn't called for guest users of the site. I wanted to support that as well and simplify the logic. So now the block has to deal with the case that `user` is `nil`.